modules/tinymce/src/core/test/ts/browser/EditorUploadTest.ts

```diff
@@ -293,6 +293,34 @@ UnitTest.asynctest('browser.tinymce.core.EditorUploadTest', (success, failure) =
       Assert.eq('Status is false', result[0].status, false);
       Assert.eq('Uri is empty', result[0].uploadUri, '');
       Assert.eq('Suitable number of stacks added', 2, editor.undoManager.data.length);
+      Assert.eq('Test', [
+        {
+          type: 'complete',
+          fragments: null,
+          content: '<p><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-mce-placeholder="" ></p>',
+          bookmark: {
+            start: [
+              0
+            ]
+          },
+          beforeBookmark: {
+            start: [
+              0
+            ]
+          }
+        },
+        {
+          type: 'complete',
+          fragments: null,
+          content: '<p></p>',
+          bookmark: {
+            start: [
+              0
+            ]
+          },
+          beforeBookmark: null
+        }
+      ] as any, editor.undoManager.data);
```

Comment:
Is there any other way to do that cast?

Reply:
It was really annoying, actually. It had two types, one which said that 'type' was required, the other that it shouldn't be there. So this one was it.